File: Iniciante/URI 1006.py
A = None
B = None
C = None
MEDIA = None

def read_numeric():
  try:
    # read for Python 2.x
    return float(raw_input())
  except NameError:
    # read for Python 3.x
    return float(input())

A = read_numeric()
B = read_numeric()
C = read_numeric()
MEDIA = (A * 2 + (B * 3 + C * 5)) / 10
print(str("MEDIA = ") + str("{:0.1f}".format(MEDIA)))

File: Iniciante/URI 1006.clj
(def a (read-string (read-line)))
(def b (read-string (read-line)))
(def c (read-string (read-line)))
(def media (/ (+ (* a 2) (* b 3) (* c 5)) 10))

(printf "MEDIA = %.1f\n" media)